Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN apt-get install -y -q --no-install-recommends \
     && apt-get -y autoclean
 
 ENV NVM_DIR /usr/local/nvm
-ENV NODE_VERSION 6.9.4
+ENV NODE_VERSION 7.9.0
```

Comment:
I bumped to 8.4, but I would not feel comfortable always pulling the latest. We should stay at one version and bump as needed so nothing breaks out of the blue. We also cannot go below 7.9 because we'd lose `async`/`await` support